src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -38,6 +37,25 @@ public static String convertWordToPigLatin(String word) {
 		return output;            
 	}
 	
+	public static List<String> convertWordFromPigLatin(String word) {		
+        List<String> outputs = new ArrayList<String>(); 
+		if (word.substring(word.length()-3, word.length()).equals("way")){			
+			outputs.add(word.substring(0, word.length()-3));
+			outputs.add("w" + word.substring(0, word.length()-3));
+		} else if (word.substring(word.length()-2, word.length()).equals("ay")){			
```

Comment:
Correct. 

Better algorithm:

* Delete "ay"
* If last letter is "w", one candidate (vowel first) is the current word with "w" removed. 
* Continue adding (consonant) candidates to the list by moving consonants from the back to the front of the word.